airflow/www/static/js/dag/details/index.tsx

```diff
@@ -138,6 +146,7 @@ const Details = ({
   const isGroupOrMappedTaskSummary = isGroup || isMappedTaskSummary;
   const showLogs = !!(isTaskInstance && !isGroupOrMappedTaskSummary);
   const showMappedTasks = !!(isTaskInstance && isMappedTaskSummary && !isGroup);
+  const showXcom = !!(isTaskInstance && !isGroupOrMappedTaskSummary);
```

Comment:
This is the same as showLogs now so perhaps we should combine them into `isIndividualTaskInstance`?

Reply:
That makes sense. Will incorporate this in.